src/components/AnonymousUsername/index.js

```diff
@@ -1,34 +1,35 @@
 import * as React from 'react';
-import {StyleSheet, Text} from 'react-native';
+import {StyleSheet, Text, TextStyle} from 'react-native';
 
 import {COLORS} from '../../utils/theme';
 import {POST_VERSION} from '../../utils/constants';
 import {fonts} from '../../utils/fonts';
 import {getOfficialAnonUsername} from '../../utils/string/StringUtils';
 
 const styles = StyleSheet.create({
-  feedUsername: {
+  feedUsername: (isFeeds) => ({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 16.94,
     color: COLORS.black,
-    flex: 1
-  }
+    flex: isFeeds ? 0 : 1
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `feedUsername` style now accepts a parameter `isFeeds` to conditionally set the `flex` property. This is a good use of conditional styling based on props.

However, it's important to note that this change could potentially affect the layout of components where `AnonymousUsername` is used. Make sure to test all instances of this component in different scenarios to ensure the layout still works as expected.

```diff
-    flex: 1
+    flex: isFeeds ? 0 : 1
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->